manifests/plugin/discovery.pp

```diff
@@ -26,6 +26,8 @@
   if $install_images {
     $tftp_root_clean = regsubst($tftp_root, '/$', '')
 
+    ensure_resource('file', "${tftp_root_clean}/boot", {ensure => directory})
+
     foreman::remote_file {"${tftp_root_clean}/boot/${image_name}":
       remote_location => "${source_url}${image_name}",
       mode            => '0644',
```

Comment:
where was the require (see line below that) satisfied before that change?

Reply:
Hidden in foreman_proxy::tftp class line 14:
`
 file { $directories:
`
I assume that we can close this PR.